packages/kiota_abstractions/lib/src/extensions/string_extensions.dart

```diff
@@ -0,0 +1,7 @@
+part of '../../kiota_abstractions.dart';
+
+extension StringExtensions on String {
+  String toFirstCharacterLowerCase() {
```

Comment:
I might have missed where this is being used. But we shouldn't need that as far as I can tell.

Reply:
Its used in `writeCollectionOfEnumValues`: https://github.com/ricardoboss/dart_kiota/pull/26/files#diff-b6ebe0b7afe1eb6dd2245d2598edf6c2aea78587617a7d27f52cbe472a9a75fcR70

Reply:
Right, do you remember where you saw that in the other languages? this is technically incorrect.
We started with that, because that convention worked well for Microsoft Graph scenarios.
But we since moved to using annotations/attributes/serialization and deserialization mapping methods in other languages so we could use the proper serialized value.

Reply:
Yes, I saw it here in the C# implementation: https://github.com/microsoft/kiota-serialization-form-dotnet/blob/50d7a29ac768c2f3c0047d8909f12c2df623a30f/src/FormSerializationWriter.cs#L99

I'm treating the C# impl as the reference implementation for all libraries. Also C# is pretty similar to Dart in most aspects.

Reply:
Thanks, I created an issue so we can address this implementation issue later on. https://github.com/microsoft/kiota-serialization-form-dotnet/issues/155